app/scripts/controllers/push-platform-notifications/types/on-chain-notification/schema.d.ts

```diff
@@ -0,0 +1,300 @@
+/**
+ * This file was auto-generated by openapi-typescript.
```

Comment:
Perhaps include one more line here on how to generate the file, maybe even just the specific command that was used?